crates/kas-widgets/src/float.rs

```diff
@@ -89,6 +89,7 @@ mod Float {
     #[widget]
     pub struct Float<C: Collection> {
         core: widget_core!(),
+        #[collection]
         widgets: C,
```

Comment:
New `#[collection]` attribute replaces two `Tile` fns and `impl Widget { ... }`.